tests/e2e/enform.spec.js

```diff
@@ -192,7 +192,13 @@ test.describe('create, use and submit EN Form', () => {
 });
 
 async function createPageWithENForm(page, admin, editor, style) {
-  await admin.createNewPost({postType: 'page', title: 'Test page with enform (style)', legacyCanvas: true});
+  //await admin.createNewPost({postType: 'page', title: 'Test page with enform (style)', legacyCanvas: true});
```

Comment:
I guess this commented line can be removed?

Reply:
Can we no longer create a page with a title directly via this function? 🤔

Reply:
Ah my bad, I changed it when I thought it was causing a crash, gonna put it back